lib/upload.rb

```diff
@@ -0,0 +1,68 @@
+# :nocov:
+require 'contentful/management'
+
+#
+# Populate Contentful using YAML content
+#
+class Upload
```

Comment:
Can your rake task and function be named to specify that it is just static pages please. this is going to collide with the version you cloned this from in my branch otherwise.